pkg/kubelet/metrics/metrics.go

```diff
@@ -162,7 +163,7 @@ var (
 			Subsystem:      KubeletSubsystem,
 			Name:           PodStartDurationKey,
 			Help:           "Duration in seconds from kubelet seeing a pod for the first time to the pod starting to run",
-			Buckets:        metrics.DefBuckets,
+			Buckets:        []float64{0.5, 1, 2, 3, 4, 5, 6, 8, 10, 20, 30, 45, 60, 120, 180, 240, 300, 360, 480, 600, 900, 1200, 1800, 2700, 3600},
```

Comment:
3600 seconds?

Reply:
yeah, the buckets here are the same as pod_start_sli_duration_seconds. We need the super long buckets because there might be node creation (in case of GKE's autoscaling, for example), stockout, retries, between a pod being created and a pod starting.  All of those could take a long time.